GameData/SPEngine/RealPlumeConfig.cfg

```diff
@@ -163,7 +163,7 @@
 	}
 }
 
-@PART[SPEngineGamma|SPEngineGamma2|SPEngineGammaAlt]:FOR[ZZREALPLUME]:NEEDS[SmokeScreen]
+@PART[SPEngineGamma|SPEngineGamma2]:FOR[ZZREALPLUME]:NEEDS[SmokeScreen]
```

Comment:
Not quite sure how this will fix anything, given that the parts that I was testing (which were plumeless) were SPEngineGamma and SPEngineGamma2.  :confused:

Reply:
Also this is identical to what feac39a had and that's what I tested already.

Reply:
well i had to add another plume for the roengine model since it has a different transform name, but yeah don't know what could be causing that, i will look into it futher

Reply:
oh i see  the issue just a min